src/js/analyses/components/Iimi/IimiDetectionTag.tsx

```diff
@@ -1,6 +1,6 @@
+import { Label } from "@base";
 import React from "react";
-import { Label } from "../../../base";
 
 export function IimiDetectionTag({ result }) {
-    return result ? <Label color="red">Detected</Label> : <Label color="grey">Undetected</Label>;
+    return result ? <Label color="red">Detected</Label> : <Label className="bg-grey">Undetected</Label>;
```

Comment:
Can't this just be like below since grey is the default?:
```suggestion
    return result ? <Label color="red">Detected</Label> : <Label>Undetected</Label>;
```